crates/edr_napi/test/files/issue-543.json

```diff
@@ -0,0 +1,582 @@
+{
```

Comment:
Given that we use the naming `data` for other folders like this (e.g. `./data` and `crates/edr_napi/data`), I would use the same name here instead of `files`.